plugins/MySQL.cpp

```diff
@@ -638,7 +649,7 @@ const char* g_MySQLVariables[MYSQL_NUM_VARIABLES][2] = {
     {"unique_checks", "ON"},
     {"updatable_views_with_limit", "YES"},
     {"version", "5.1.73-log"},
-    {"version_comment", SVERSION},
+    {"version_comment", SVERSION},//SVERSION},
```

Comment:
did you mean to revert this previous change (to (Ubuntu))? in which case, remove the useless comment?

Reply:
Yeah, I meant to revert it. Turns out this change isn't necessary, just the one on line 80. One more change coming.